src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources
```

Comment:
I actually meant `<resources translatable="false">`, which means this will be ignored by Transifex (and should also ignore the `MissingTranslation` warning).